modules/nf-core/trust4/main.nf

```diff
@@ -13,13 +13,15 @@ process TRUST4 {
     tuple val(meta3), path(vdj_reference)
 
     output:
-    tuple val(meta), path("*.tsv")          , emit: tsv
-    tuple val(meta), path("*_airr.tsv")     , emit: airr_tsv
-    tuple val(meta), path("*_report.tsv")   , emit: report_tsv
-    tuple val(meta), path("*.fa")           , emit: fasta
-    tuple val(meta), path("*.out")          , emit: out
-    tuple val(meta), path("*.fq")           , emit: fq
-    path "versions.yml"                     , emit: versions
+    tuple val(meta), path("*.tsv")                          , emit: tsv
```

Comment:
Is there a reason the emit statements are so far out? You should align them to the end of the longest line if not